### PR TITLE
[zh] Sync release 1.19 release announcement and upgrade notes into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
Translate release 1.19 release announcement and upgrade notes into Chinese:

```
content/zh/docs/releases/supported-releases/index.md
content/zh/news/releases/1.19.x/_index.md
content/zh/news/releases/1.19.x/announcing-1.19/_index.md
content/zh/news/releases/1.19.x/announcing-1.19/upgrade-notes/index.md
```

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/news/releases/1.19.x/announcing-1.19/_index.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+### Gateway API
+
+Kubernetes [Gateway API](http://gateway-api.org/)
+是一项旨在为 Kubernetes 带来丰富的服务网络 API
+（类似于 Istio VirtualService 和 Gateway）的举措。
+
+在此版本中，配合 Gateway API v0.8.0 版本，
+正式添加了[服务网格支持](https://gateway-api.sigs.k8s.io/blog/2023/0829-mesh-support/)！
+这项工作是跨越更广泛的 Kubernetes 生态系统的社区努力，并且包含多个一致的实现（包括 Istio）。
```
Comment (windsonsea) [inline content/zh/news/releases/1.19.x/announcing-1.19/_index.md:37]:
```suggestion
这项进展是与更广泛的 Kubernetes 生态社区共同努力的结果，并且包含 Istio 在内的多个合规性实现。
```

File: content/zh/news/releases/1.19.x/announcing-1.19/_index.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+查看[网格文档](/zh/docs/tasks/traffic-management/ingress/gateway-api/#mesh-traffic)以开始使用。
+与任何实验性功能一样，我们非常感谢反馈。
+
+除了网状流量之外，入口流量的 API
```
Comment (windsonsea) [inline content/zh/news/releases/1.19.x/announcing-1.19/_index.md:42]:
```suggestion
除了网格流量之外，入口流量的 API
```

**Review by windsonsea — APPROVED** [review]

**Comment by wilsonwu** [comment]
Thanks @windsonsea I will apply the suggestions in another sync PR for 1.19 change notes.

**Comment by windsonsea** [comment]
oh... clicked a wrong option when submitting the comments....

**Pull request merged**